### DISCUSSION TITLE
passwall2 可以使用 mosdns之类的第三方DNS吗？

### DISCUSSION THREAD

**Discussion by LitCcc** [Openwrt-Passwall/openwrt-passwall2#20] [Q&A] [answered]
![Snipaste_2022-03-17_10-30-17](https://user-images.githubusercontent.com/36120072/158724591-8cc8d008-0de8-40dc-a7b8-366a31e8218e.png)

passwall 中可以在配置好 mosdns 之后把dns的过滤模式设置为`通过UDP请求DNS` 并设置远程地址为需要转发的DNS服务器，passwall2中好像去掉了这个选项

**Comment by davidtall** [comment]
https://github.com/QiuSimons/openwrt-mos 用这个

**Comment by handsomeji** [comment, marked as answer]
可以，但不建議。出現性能問題無法解決。
